netcanv-renderer-opengl/src/font.rs

```diff
@@ -115,19 +149,22 @@ impl FontFace {
       );
    }
 
-   fn glyph_renderer(&mut self, size: u32) -> GlyphRenderer<'_, '_, '_> {
+   fn glyph_renderer(&mut self, size: u32) -> GlyphRenderer<'_, '_, '_, '_> {
       self.make_size(size);
+
       GlyphRenderer {
-         face: &self.face,
+         swash_font: self.swash_font.as_ref(),
          gl: &self.gl,
          size_store: self.sizes.get_mut(&size).unwrap(),
+         scale_context: &mut self.scale_context,
+         shape_context: &mut self.shape_context,
       }
    }
 }
 
 impl Drop for FontFace {
    fn drop(&mut self) {
-      for size in self.sizes.values() {
+      for (_, size) in &self.sizes {
```

Comment:
Why change this iterator? Clippy will emit a warning that `.values()` is the preferred way of iterating over `HashMap` values.